packages/opentelemetry-api/src/api/trace.ts

```diff
@@ -81,4 +83,8 @@ export class TraceAPI {
   public disable() {
     delete _global[GLOBAL_TRACE_API_KEY];
   }
+
+  public isSpanContextValid(sc : SpanContext) : boolean {
+    return isSpanContextValid(sc);
+  }
```

Comment:
You can prevent unnecessary function overhead by doing:

```suggestion
  public isSpanContextValid = isSpanContextValid;
```